### PR TITLE
L10n us administrative

### PR DESCRIPTION
**Pull request by bwrsandman**
Transfer of https://github.com/OCA/account-invoice-reporting/pull/30 to right repo

Feel free to clone this PR and work on it.

Please follow the official guide from the [OCA Guidelines page](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md) for your review.

### CODE REVIEW

**Comment by bwrsandman** [comment]
Company logos should be removed

File: l10n_us_administrative/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "summary": """Add missing states, add US county
+equivalents with filter (Ursa).""",
+    'description': """Based on US Census information (updated 2014)
+
+* adds a dropdown list on the partner form view, filtered by state.
+
+In some states, sales tax rates are based on Counties.
+For those where this is the case, this module could be used in
+conjunction with rules to determine sales tax.""",
+    'maintainer': 'Ursa Information Systems',
```
Comment (bwrsandman) [inline l10n_us_administrative/__openerp__.py:22]:
```
'maintainer': 'Ursa Information Systems,'
              'Odoo Community Association (OCA)'
```

**Comment by bwrsandman** [comment]
This PR is here to be worked on. Feel free to fork this branch if you like the module. I don't think I will be working on it.

File: l10n_us_administrative/static/description/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">United States - States & Counties</h2>
```
Comment (gurneyalex) [inline l10n_us_administrative/static/description/index.html:4]: syntax error: replace '&' with '&amp;'

**Pull request merged**